Demo/src/debug/AndroidManifest.xml

```diff
@@ -12,7 +12,8 @@
         <meta-data android:name="com.google.android.gms.wallet.api.enabled" android:value="true" />
 
         <activity
-            android:name=".MainActivity">
+            android:name=".MainActivity"
+            android:exported="true">
```

Comment:
Merchants need to add this to their apps right for Android 12 support? Did we call it out in all the right places? (Changelog, README, Docs)?

Reply:
So merchants don't actually need to make this change until they update their own apps to target/compile for API 31 (Android 12). If they use this version in an app that doesn't target Android 12, the exported tags are still optional. They would need to add the exported flags when they update their target/compile version, but I think that's probably outside the scope of our CHANGELOG right? We did update our [docs examples](https://developer.paypal.com/braintree/docs/guides/client-sdk/setup/android/v4#browser-switch-setup) to include this though.